common/changes/office-ui-fabric-react/createBaseWellPickerAndBaseFloatingPicker_2017-10-05-17-24.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Why does Fabric need so many change files?

Reply:
Looking at other PRs it looks like only the ones for the changed packages should be commit, although one for each package is created. I'll remove the unneeded files in my next commit :)